<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="WINDOWS-NBFRO6B" failures="1" tests="1" name="Default test(failed)" time="4.295" errors="0" timestamp="2023-07-21T13:22:56 IST">
  <testcase classname="api.test.UserTests" name="testDeleteUser" time="3.819">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testDeleteUser(UserTests.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUser -->
  <testcase classname="api.test.UserTests" name="testgetUser" time="0.0">
    <ignored/>
  </testcase> <!-- testgetUser -->
  <testcase classname="api.test.UserTests" name="testPostUser" time="0.0">
    <ignored/>
  </testcase> <!-- testPostUser -->
  <testcase classname="api.test.UserTests" name="testUpdateUser" time="0.0">
    <ignored/>
  </testcase> <!-- testUpdateUser -->
</testsuite> <!-- Default test(failed) -->
